wrangler-core/src/test/java/co/cask/wrangler/steps/SplitEmailTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.wrangler.steps;
+
+import co.cask.wrangler.api.Record;
+import co.cask.wrangler.api.StepException;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Tests {@link SplitEmail}
+ */
+public class SplitEmailTest {
+  @Test
+  public void testSplitEmail() throws Exception {
+    String[] directives = new String[] {
+      "split-email email",
+    };
+
+    List<Record> records = Arrays.asList(
+      new Record("email", "root@cask.co"),
+      new Record("email", "joltie.xxx@gmail.com"),
+    new Record("email", "joltie_xxx@hotmail.com")
```

Comment:
can you add a test for an email that looks like this: a."b@c".d@e.f   after you fix the parsing above?